### PR TITLE
Test on oldest supported SDK

### PR DESCRIPTION
**Pull request by kevmoo**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,9 +1,9 @@
 language: dart
 dart:
+  - 2.0.0
```
Comment (devoncarew) [inline .travis.yml]: why not `stable` here?

File: .gitignore [hunk 2]
```diff
@@ -1,11 +1,3 @@
-.buildlog
```
Comment (devoncarew) [inline .gitignore]: can you revert the changes to this file?

Comment (kevmoo) [reply .gitignore]: No. 😄

File: .travis.yml (hunk 1 above)
Comment (kevmoo) [reply .travis.yml]: Want to test on the oldest supported SDK. We've been burned here too many times.

File: .gitignore (hunk 2 above)
Comment (kevmoo) [reply .gitignore]: I really want our repos to show best practices. There is no need to ignore any of those other files/dirs

Comment (kevmoo) [reply .gitignore]: cool @devoncarew ?
